packages/sandcastle/src/InfoBadge.css

```diff
@@ -0,0 +1,8 @@
+.info-badge {
+  display: inline-block;
+
+  .🥝-label > & {
```

Comment:
Is text replacement from trying the write `kiwi` referring to the library. This should be changed, right?